dts/arm/atmel/sam3x.dtsi

```diff
@@ -49,14 +51,16 @@
 			compatible = "atmel,sam-usart";
 			reg = <0x4009C000 0x130>;
 			interrupts = <18 0>;
+			pid = <18>;
 			status = "disabled";
 			label = "USART_1";
 		};
 
 		usart2: usart@400A0000 {
 			compatible = "atmel,sam-usart";
 			reg = <0x400A0000 0x130>;
-			interrupts = <18 0>;
+			interrupts = <19 0>;
+			pid = <19>;
```

Comment:
do you mind either putting the fix for uart in a separate commit, or at least comment in the commit message about fixing this.

Reply:
Sorry about that, I put it now in a separate commit. s/pid/peripheral-id/g is done too.